### PR TITLE
Azure - Add basic ServerGroup and SecurityGroup creation

### PR DESCRIPTION
**Pull request by JargoonPard**
Simplistic ServerGroup creation is in place but without instance monitoring
SecurityGroup creation and viewing is in place.

### CODE REVIEW

File: app/scripts/modules/azure/securityGroup/configure/CreateSecurityGroupCtrl.js [hunk 1]
```diff
@@ -21,29 +36,147 @@ module.exports = angular.module('spinnaker.azure.securityGroup.create.controller
...
+        ruleset[index].destinationPortRange =
+            ruleset[index].startPort + '-' + ruleset[index].endPort;
+    };
+
+    ctrl.removeRule = function(ruleset, index) {
+      ruleset.splice(index, 1);
+    };
+
+    ctrl.moveUp = function(ruleset, index) {
+      if(index == 0)
```
Comment (anotherchrisberry) [inline app/scripts/modules/azure/securityGroup/configure/CreateSecurityGroupCtrl.js]: minor formatting: `if(index == 0)` should be `if (index === 0)`

File: app/scripts/modules/azure/securityGroup/configure/CreateSecurityGroupCtrl.js [hunk 2]
```diff
@@ -21,29 +36,147 @@ module.exports = angular.module('spinnaker.azure.securityGroup.create.controller
...
+
+          return azureSecurityGroupWriter.upsertSecurityGroup($scope.securityGroup, application, 'Create', params);
+        }
+      );
+    };
+
+    ctrl.addRule = function(ruleset) {
+      ruleset.push({
+        name: $scope.securityGroup.name + '-Rule' + ruleset.length,
+        priority: ruleset.length == 0 ? 100 : 100 * (ruleset.length + 1),
```
Comment (anotherchrisberry) [inline app/scripts/modules/azure/securityGroup/configure/CreateSecurityGroupCtrl.js]: `==` -> `===`

File: app/scripts/modules/azure/securityGroup/configure/CreateSecurityGroupCtrl.js [hunk 3]
```diff
@@ -21,29 +36,147 @@ module.exports = angular.module('spinnaker.azure.securityGroup.create.controller
...
+        sourcePortRange: '*',
+        destinationAddressPrefix: '*',
+        destinationPortRange: '7001-7001',
+        startPort: 7001,
+        endPort: 7001
+      });
+    };
+
+    ctrl.portUpdated = function(ruleset, index)
+    {
```
Comment (anotherchrisberry) [inline app/scripts/modules/azure/securityGroup/configure/CreateSecurityGroupCtrl.js]: Brace style is a little inconsistent - not breaking anything, just something that's easier to clean up and standardize on earlier than later.

I will probably add an eslint rule later to warn against non one-true-brace-style braces, but won't prevent code compilation.

**Comment by anotherchrisberry** [comment]
There are a few minor style and formatting things, but I'm happy to merge this if you're good with it.

**Comment by JargoonPard** [comment]
Thanks @anotherchrisberry.  I'll get those items cleaned up on the next PR if you would merge this PR.  Thanks!

**Pull request merged**